dev-python/pybtex/pybtex-0.22.2.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="BibTeX-compatible bibliography processor"
+HOMEPAGE="https://pybtex.org https://pypi.org/project/pybtex/"
+SRC_URI="https://files.pythonhosted.org/packages/source/p/${PN}/${P}.tar.gz"
+PATCHES=( "${FILESDIR}/${P}-fix-test-installation.patch" )
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/latexcodec[${PYTHON_USEDEP}]
+		dev-python/pyyaml[${PYTHON_USEDEP}]
+		dev-python/six[${PYTHON_USEDEP}]"
+
+distutils_enable_tests pytest
```

Comment:
```
===================================== short test summary info ======================================
ERROR tests/plugin_test.py - ModuleNotFoundError: No module named 'nose'
ERROR tests/plugin_test.py
ERROR tests/richtext_test.py - ModuleNotFoundError: No module named 'nose'
ERROR tests/richtext_test.py
ERROR tests/database_test/database_test.py - ModuleNotFoundError: No module named 'nose'
ERROR tests/database_test/database_test.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 6 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================== 12 warnings, 6 errors in 1.57s ==================================
```

Worked after installing `dev-python/nose`.

Reply:
Moved dependencies to RDEPEND, added RDEPEND and dev-python/nose to BDEPEND conditional on test use flag.